spec/urn_spec.rb

```diff
@@ -2,51 +2,61 @@
 require 'urn'
 
 RSpec.describe URN do
-  describe '#valid?' do
-    it 'returns true if it is valid' do
-      expect(described_class.new('urn:namespace:specificstring')).to be_valid
+  describe 'Kernel.URN' do
```

Comment:
Actually, it comes from `main`, a weird instance of `Object`. Interesting article: https://banisterfiend.wordpress.com/2010/11/23/what-is-the-ruby-top-level/